portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,15 +3,17 @@
 import com.google.appengine.api.datastore.Entity;
 
 public final class Comment {
+  private final String author;
   private final String value;
   private final long timeMillis;
 
-  public Comment(String comment, long time) {
+  public Comment(String name, String comment, long time) {
+    this.author = name;
     this.value = comment;
     this.timeMillis = time;
   }
 
   public static Comment fromEntity(Entity comment) {
-    return new Comment((String) comment.getProperty("value"), (Long) comment.getProperty("timeMillis"));
+    return new Comment((String) comment.getProperty("name"), (String) comment.getProperty("value"), (Long) comment.getProperty("timeMillis"));
```

Comment:
Try to [keep lines under 100 characters](https://stackoverflow.com/q/3434367) so readers don't have to scroll sideways to see everything.

Reply:
So after reading through that post, I realized I didn't need to store the properties as their actual types in Comment because it all becomes JSON strings anyways. Then I realized I could take it a step further and instead of converting Entity->Comment->JSON I could just go straight from Entity->JSON without any trouble, so that's the implementation I went with in the last 2 commits.